pymtl3/stdlib/test/src_sink_test.py

```diff
@@ -199,3 +199,67 @@ def test_adaptive( src_level, sink_level, msgs, src_init,  src_intv,
                     src_init,  src_intv, sink_init,
                     sink_intv, arrival_time )
   th.run_sim()
+
+#-------------------------------------------------------------------------
+# Error message test
+#-------------------------------------------------------------------------
+
+def test_error_more_msg():
+  try:
+    th = TestHarnessSimple(
+      Bits16, TestSrcCL, TestSinkCL,
+      src_msgs  = [ b16(0xface), b16(0xface) ],
+      sink_msgs = [ b16(0xface) ],
+    )
+    th.run_sim()
+    errored = False
```

Comment:
You don't need this variable. Just return in the except block and raise another exception instead of assert errored.